website/src/models/user.js

```diff
@@ -688,7 +688,6 @@ mongoose.model('User')
   .then((foundMods) => {
     // Using push to maintain the reference to mods
     mods.push(...foundMods);
-  })
-  .catch((err) => {
+  }, (err) => {
```

Comment:
@crookedneighbor would you mind adding a TODO comment here about replacing this with a catch in future? The build is stuck but otherwise works well!